bridge/ffi/src/c_api.rs

```diff
@@ -414,6 +414,33 @@ pub unsafe extern "C" fn graph_get_public_keys(
 	})
 }
 
+/// Returns the deserialized dsnp keys
+#[no_mangle]
+pub unsafe extern "C" fn graph_deserialize_dsnp_keys(
```

Comment:
since the `free_graph_dsnp_public_keys` already exist don't need to add a new one